### PR TITLE
Fix build issues due to release new visual studio / dotnet core 2.0

### PR DESCRIPTION
**Pull request by logophobia**

### CODE REVIEW

File: CollAction/CollAction.csproj [hunk 1]
```diff
@@ -20,27 +21,7 @@ npm run build</PreBuildEvent>
   </PropertyGroup>
 
   <ItemGroup>
-    <Content Include="wwwroot\images\team\Brian Russell.png" />
```
Comment (stewartmatheson) [inline CollAction/CollAction.csproj]: These where causing the build to break for me. I have already deleted them from my working copy.

File: CollAction/Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-﻿FROM microsoft/aspnetcore:1.1.1
+﻿FROM microsoft/aspnetcore:1.1.2
```
Comment (stewartmatheson) [inline CollAction/Dockerfile]: So we still need to build first on the host OS which is causing some issues. Is there an easy way to get a dev server working within docker too?

File: docker-compose.vs.debug.yml [hunk 3]
```diff
@@ -1,20 +0,0 @@
-version: '2'
```
Comment (stewartmatheson) [inline docker-compose.vs.debug.yml]: How come we dont need this any more?

File: docker-compose.dcproj [hunk 4]
```diff
@@ -5,17 +5,13 @@
     <DockerLaunchBrowser>True</DockerLaunchBrowser>
```
Comment (stewartmatheson) [inline docker-compose.dcproj]: Is this file just for visual studio? Is there a way to run this from the command line tools?

File: CollAction/Dockerfile (hunk 2 above)
Comment (logophobia) [reply CollAction/Dockerfile]: No easy, but maybe we should make a build docker image

File: docker-compose.vs.debug.yml (hunk 3 above)
Comment (logophobia) [reply docker-compose.vs.debug.yml]: It's been replaced by version: '3'

The whole update process is something visual studio does automatically.

File: docker-compose.dcproj (hunk 4 above)
Comment (logophobia) [reply docker-compose.dcproj]: It's a visual studio project to run that enables you to run it in docker automatically.

**Pull request merged**